examples/wrap/wrap_test.c

```diff
@@ -725,21 +725,21 @@ int TPM2_Wrapper_Test(void* userCtx)
     /*------------------------------------------------------------------------*/
     XMEMSET(&aesKey, 0, sizeof(aesKey));
     rc = wolfTPM2_LoadSymmetricKey(&dev, &aesKey, TEST_AES_MODE,
-        (byte*)kTestAesCbc128Key, (word32)XSTRLEN(kTestAesCbc128Key));
+        (byte*)TEST_AES_KEY, (word32)XSTRLEN(TEST_AES_KEY));
```

Comment:
Using XSTRLEN here and below at 735 and 742 isn't going to work for either case (CFB or CBC)... one is string and one is byte array. Let's change them all to byte arrays and use sizeof().